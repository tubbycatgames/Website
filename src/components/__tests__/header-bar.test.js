import React from "react"
import renderer from "react-test-renderer"

import HeaderBar from "../header-bar"

describe("Header Bar", () => {
  it("renders the full header bar", () => {
    const tree = renderer.create(<HeaderBar />)
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
