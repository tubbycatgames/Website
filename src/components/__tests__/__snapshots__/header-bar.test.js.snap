// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Bar renders the full header bar 1`] = `
<div
  className="header"
>
  <div
    className="icons"
  >
    <a
      href="https://tubbycatgames.com"
      onClick={[Function]}
    >
      <img
        alt="Tubby Cat Games Icon"
        className="logo"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://tubbycatgames.itch.io/"
      onClick={[Function]}
    >
      <img
        alt="Itch Page Icon"
        className="icon"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://gamejolt.com/@tubbycatgames"
      onClick={[Function]}
    >
      <img
        alt="Game Jolt Page Icon"
        className="icon"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://github.com/tubbycatgames"
      onClick={[Function]}
    >
      <img
        alt="GitHub Page Icon"
        className="icon"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://discord.gg/vyh5XY"
      onClick={[Function]}
    >
      <img
        alt="Discord Social Media Icon"
        className="icon"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://twitter.com/tubbycatgames"
      onClick={[Function]}
    >
      <img
        alt="Twitter Social Media Icon"
        className="icon"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://instagram.com/tubbycatgames"
      onClick={[Function]}
    >
      <img
        alt="Instagram Social Media Icon"
        className="icon"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://fb.me/tubbycatgames"
      onClick={[Function]}
    >
      <img
        alt="Facebook Social Media Icon"
        className="icon"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://www.youtube.com/channel/UC_6l3SdhVjdtTd2DnHK2JbQ"
      onClick={[Function]}
    >
      <img
        alt="YouTube Channel Icon"
        className="icon"
        src="test-file-stub"
      />
    </a>
  </div>
  <div
    className="links"
  >
    <a
      href="/"
    >
      <p>
        About
      </p>
    </a>
    <a
      href="/games"
    >
      <p>
        Games
      </p>
    </a>
    <a
      href="/blog"
    >
      <p>
        Blog
      </p>
    </a>
  </div>
</div>
`;
